src/algebra/pi_instances.lean

```diff
@@ -241,6 +241,14 @@ lemma fst.is_monoid_hom [monoid α] [monoid β] : is_monoid_hom (prod.fst : α 
 lemma snd.is_monoid_hom [monoid α] [monoid β] : is_monoid_hom (prod.snd : α × β → β) :=
 { map_mul := λ _ _, rfl, map_one := rfl }
 
+@[to_additive prod.add_monoid_hom.fst]
```

Comment:
Probably this is fine, but do you really need to give an explicit name for `to_additive` here?

Reply:
Probably yes, because otherwise you end up with `sum.add_monoid_hom.fst` which is not what you want.

Reply:
Ah!